tests/run/test_patma.py

```diff
@@ -0,0 +1,3415 @@
+### COPIED FROM CPython 3.9
```

Comment:
Probably worth updating from latest CPython main branch already.
https://github.com/python/cpython/blob/main/Lib/test/test_patma.py

Reply:
I'll update it in the next update although the difference is very small. The comment was clearly wrong since it's a CPython 3.10 feature anyway.